Screens/Home.tsx

```diff
@@ -1,27 +1,58 @@
-import { View, Text, Button, TouchableOpacity } from "react-native";
-import React from "react";
+import { View, Text, Button, TouchableOpacity, Modal, TextInput } from "react-native";
+import { useContext, useState } from "react";
 import { useNavigation } from "@react-navigation/native";
-import styles from "../Styles/styleconfig";
-import Header from "../components/header";
-import MadeButton from "../components/button";
+import { RedButton, Header, GreenButton } from "../components";
+import ThemeContext from "../theme/ThemeContext";
+import { LightStyles, DarkStyles } from "../Styles";
+import { Ionicons } from "@expo/vector-icons";
 
 const Home = () => {
+  const { theme } = useContext(ThemeContext);
+  const styles = theme === "light" ? LightStyles : DarkStyles;
   const nav = useNavigation();
 
-  const handlerSettings = () => {
-    nav.navigate("Settings" as never);
-  };
+  //state to check if modal is visible
+  const [isModalVisible, setModalVisible] = useState(false);
 
-  return (
-    <View style={styles.container}>
-      <Header headerTitle={"Home"} />
-      <MadeButton
-        onPress={handlerSettings}
-        disabled={false}
-        text={"Go to Settings"}
+  //set modal to visible -> True
```

Comment:
Toggle function would possibly be better than 2 separate states that always turns to opposite of the current state